src/main/resources/view/MainWindow.fxml

```diff
@@ -41,7 +41,7 @@
         </StackPane>
 
         <StackPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="pane-with-border"
-                   minHeight="100" prefHeight="100" maxHeight="100">
+                   minHeight="100" prefHeight="150" maxHeight="150">
```

Comment:
Might have to see if the window is minimized it will still show the entire message